package.json

```diff
@@ -11,33 +11,23 @@
     "test": "NODE_ENV=test mocha && echo 'Tests against app mounted at /feedback:' && EXPRESS_BASE_PATH=/feedback NODE_ENV=test mocha",
     "db-migrate": "knex migrate:latest --knexfile db/knexfile.js",
     "db-rollback": "knex migrate:rollback --knexfile db/knexfile.js",
-    "heroku-postbuild": "npm run compile",
+    "heroku-postbuild": "npm install --dev && npm run compile && npm prune --production",
```

Comment:
There was an issue with heroku deployment. According to heroku [documentation](https://devcenter.heroku.com/articles/nodejs-support#devdependencies) all of the npm commands are run with `NODE_ENV=production`